### ISSUE TITLE
interested in the method of training re-lighting model

### ISSUE THREAD

**Issue by feiyangsuo** [lllyasviel/IC-Light#44]
Hi, thanks for the great work!
I'm curious about how is a model like this trained to achieve re-lighting ability. Is there any plan of a paper release?

**Comment by Husky110** [comment]
@feiyangsuo - afaik the code uses a standard SD 1.5 model for it's stuff...

**Comment by feiyangsuo** [comment]
> @feiyangsuo - afaik the code uses a standard SD 1.5 model for it's stuff...

sure, so I suppose the magic lies in the training method (or training data perhaps)?

**Comment by Husky110** [comment]
As far as I can see that magic lies in everything BUT the model... (Minus gradio of course...)

**Comment by caojiehui** [comment]
Hello, I appreciate the excellent work!
Are there plans to release the training code and data?

**Comment by bhosalems** [comment]
Yes looking forward to the release of the training code since the paper has already been accepted.

**Comment by bhosalems** [comment]
yes this is the paper.
